packages/lib/src/h5web/toolbar/controls/DomainSlider/Histogram.tsx

```diff
@@ -22,8 +23,10 @@ function Histogram(props: Props) {
   const { values, bins, scaleType, sliderDomain, dataDomain } = props;
   const { colorMap, invertColorMap } = props;
 
-  const binDomain = useDomain(bins);
-  const xDomain = useCombinedDomain([binDomain, sliderDomain, dataDomain]);
+  const binDomain = useDomain(bins, scaleType) || DEFAULT_DOMAIN;
```

Comment:
1. The `binDomain` could be unsupported by the current `scaleType`. This is fixed by taking `scaleType` into account when computing `binDomain`